Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2-slim
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user && mkdir -p /code && chown user:user /code
+
+RUN apt-get update && apt-get install -y --no-install-recommends \

```

Comment:
If we copy the source from our local version, we could skip installing git, cloning the respository and uninstalling git again:

`COPY . /code`